e2e/tests/integrations/request-activation-as-integrator.test.ts

```diff
@@ -0,0 +1,78 @@
+import { expect, test } from "@playwright/test";
+import { fakerNL_BE as faker } from "@faker-js/faker";
+
+test("As an integrator I can request activation via organization details", async ({
+  page,
+}) => {
+  // Create SAPI integration
+  await page.goto("/nl/integraties");
```

Comment:
I guess it can be useful to make this a reusable method?

Reply:
The creation of an integration you mean?
Yes I was thinking of the same thing